src/constants.ts

```diff
@@ -23,3 +23,4 @@ export const SETTINGS = {
   RAVEN_DSN: '',
   Raven: undefined
 }
+export const GLOBAL_PLUGIN_NAME = '_udplugins'
```

Comment:
plugin constructor stored in window['_udplugins'] when call 'provide'

Reply:
😰